addons/binding/org.openhab.binding.upb/OSGI-INF/UPBHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
It seems that this file has a missing  or invalid copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.